src/Search/SearchParametersFilter.php

```diff
@@ -40,6 +40,7 @@ class SearchParametersFilter
         self::TYPE_CITIZEN_ACTIONS,
     ];
 
+    const RADIUS_ALL = 0;
```

Comment:
What about the `RADIUS_NONE = -1` ?

Reply:
"None" and "all" are the same in this context, I prefer "ALL" though.

Reply:
I don't think so, -1 is for user who don't want to saw citizen project. ping @thomasbisignani ?

Reply:
Ok for `-1` ! Thx.